hw/top_earlgrey/data/ip/chip_uart_testplan.hjson

```diff
@@ -0,0 +1,252 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_uart
+  testpoints: [
+    // Basic smoke tests:
+    {
+      name: chip_sw_uart_tx_rx_smoke
+      desc: '''Verify the transfer of data from UART
+
+               - Powerup OT and enable UART IO.
+               - Calculate CONTROL.NCO using programmers model and update the register
```

Comment:
Nit: I think you probably want "programmer's" here (and on similar lines below).

A quick grep in the tree shows that we haven't always put apostrophes in properly, but we may as well get it right here :-)

Reply:
Updated apostrophe

Reply:
I'm really sorry, but I think we just lost a vowel!